### ISSUE TITLE
Warn about `IV.fromLength` and `Key.fromLength` not being cryptographically secure generators

### ISSUE THREAD

**Issue by InnovativeInventor** [leocavalcante/encrypt#246]
Keys should be generated randomly, and IVs should be generated randomly. So, a reasonable user would think that `IV.fromLength(n)` and `Key.fromLength(n)` would produce random keys and IVs of length `n`. This does not happen. Instead `fromLength` produces keys and IVs initialized with zeros. **NONE** of the symmetric examples in the README are secure by default. This causes issues in many downstream packages. This should be addressed immediately.

**Comment by leocavalcante** [comment]
`fromLength` are never meant to be cryptographically secure pseudorandom number generators, I think people know that. They are only for examples.

I can left this issue to improve the docs at README.
(or feel free to make a contribution)

**Comment by InnovativeInventor** [comment]
> `fromLength` are never meant to be cryptographically secure pseudorandom number generators, I think people know that. They are only for examples.

How would they know that? This is not stated anywhere in the README. Additionally, I scanned all downstream packages that rely on `encrypt`. A surprising number of downstream users use `IV.fromLength` in production. I've filed this issue because I've already begun the process of notifying them. The fact that many downstream users use `fromLength` in this insecure manner is indicative of a problem with the library.

> I can left this issue to improve the docs at README.

Sure, but I'd argue that a good crypto library should not allow users to shoot themselves in the foot and should absolutely document this behavior! In what world would a user *want* to set the `IV` to 0?

>  (or feel free to make a contribution)

I'll probably open up a pull request, then.
